A3ServerTool/Resources/Lang.xaml

```diff
@@ -286,6 +286,9 @@
     <v:String x:Key="LocalizableMaxMemoryTitle">Memory Allocation Limit</v:String>
     <v:String x:Key="LocalizableMaxMemoryTooltip">Overrides memory allocation limit to a certain amount (in megabytes). Be aware that 32-bit(x86) executable and Windows OS have significantly lower memory ceiling than both 64-bit executable and OS.</v:String>
 
+    <v:String x:Key="LocalizableLimitFpsTitle">Limit FPS</v:String>
+    <v:String x:Key="LocalizableLimitFpsTooltip">Overrides default server fps limit.</v:String>
```

Comment:
Should there be some mention of performance/profiling binary here?

Reply:
Yes, i think it should be here since it will cause some confusion without it. We will remove this mention when fps unlocker startup parameter will make it into stable Arma branch 😄